src/licensedcode/data/licenses/cal-software.yml

```diff
@@ -0,0 +1,10 @@
+key: cal-software
```

Comment:
Can you add the version to this and rename that to this key and file names to `corporate-accountability-1.1`?
There is another unrelated OSI-approved license called cal-1.0.